### ISSUE TITLE
<publicationStatus>

### ISSUE THREAD

**Issue by SJagodzinski** [SAA-SDT/eac-cpf-schema#85]
## Publication Status
- remove the element from EAC-CPF
- **add new optional attribute** `@publicationStatus` with restricted values (approved, inProcess, published) to `<control>` instead

## Creator of issue
1. Silke Jagodzinski
2. TS-EAS: EAC-CPF subgroup
3. silkejagodzinski@gmail.com

## Related issues / documents
[`<publicationStatus>`: element content should become attribute value #6](https://github.com/SAA-SDT/eac-cpf-schema/issues/6)
`<agentType>` #94 

## EAD3 Reconciliation
**Additional EAD 3 attributes**
`@altrender` - Optional
`@audience` - Optional (values limited to: external, internal)
`@encodinganalog` - Optional
`@lang` - Optional
`@script` - Optional

**May contain**: [text]

## Context
The current publication status of the EAC-CPF instance.

Revising EAC-CPF, the element will be removed and an attribute <control @publicationStatus> will indicate the current publication status of the EAC-CPF instance.

## Solution documentation
- remove from Tag Library

**Comment by SJagodzinski** [comment]
EAC-CPF Berlin meeting, 11 March 2020:

Use `<publicationstatus>` as an empty element without text or other content with a required attribute `@value` with limited values and without language attributes. This element is optional and not repeatable.

**Comment by ailie-s** [comment]
Tag Library text:

**Summary**: An optional child element of `<control>` that records the current publication status of the EAC-CPF instance.
**May contain**: [empty]
**May occur within**: `<control>`
**Description and Usage**: Use `<publicationStatus>` to indicate the current publication status of the EAC-CPF instance, whether in process, approved, or published. An optional element, `<publicationStatus>`, is only necessary if it supports local maintenance practice.
If present, the current publication status must always be reflected in the required `@value` attribute. The element itself must remain empty.
**Attributes**:
`@audience` - optional (values limited to: external, internal)
`@id`- optional
`@value` - required (values limited to: inprocess, approved, published)
**Attribute Usage**: Use `@value`, which offers a controlled list of terms, to provide information about the current publication status of the EAC-CPF instance.
**Availability**: Optional, not repeatable

**Comment by SJagodzinski** [comment]
Outcome EAC-CPF September 2020 meeting:

We will not have control type elements anymore; we will move them to attributes to their parent elements.
We have decided to put the elements `<maintenanceStatus>` and `<publicationStatus>` as attributes into `<control>`.

**Comment by kerstarno** [comment]
Schema team, schema tests:

I've tested the draft XSD and RNG schemas for EAC-CPF 2.0 with regard to "publicationStatus". There is no option anymore to include an element `<publicationStatus>` anywhere within `<control>`, but the attribute `@publicationStatus` is now being made available instead for `<control>`. All existing values of the element have been moved to the attribute.

Schema test successful.
